ios/FluentUI/Table View/TableViewCellTokenSet.swift

```diff
@@ -81,10 +81,16 @@ public class TableViewCellTokenSet: ControlTokenSet<TableViewCellTokenSet.Tokens
         super.init { token, theme in
             switch token {
             case .backgroundColor:
-                return .dynamicColor { theme.aliasTokens.colors[.background1] }
+                return .dynamicColor {
+                    DynamicColor(light: theme.aliasTokens.colors[.background1].light,
```

Comment:
why do we need this change? seems like both light and dark are coming from .backgoroudn1?